packages/engine/tests/units/state/empty/array/src/behaviors/test.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Verifies the type of the list
+ */
+const behavior = (state, context) => {
+  state.l1_is_list = typeof state.l1 === "object";
```

Comment:
Does `Array.isArray` work?

Reply:
I changed the check to `Array.isArray(state.l1)`